### ISSUE TITLE
Rewrite Installfest starting from current upstream docs

### ISSUE THREAD

**Issue by decklin** [railsbridge-boston/docs#47]
Here's upstream's installfest page:

http://docs.railsbridge.org/installfest/

I think this is a lot clearer for newbies, and has a more coherent sense of linearity (step 1, step 2, step 3...) it also asks you to choose your OS up front to avoid distracting lists of every-possible-OS options. I want to use this as a starting point but add the VM stuff to it.

Let me know if you have thoughts or ideas about what to include, or if there's something we don't like about the upstream instructions (other than RailsInstaller/RVM :sunglasses:).

**Comment by decklin** [comment]
Another thing to think about: this has them actually create a throwaway app as part of the installfest. I think this is a good idea for most students -- they have the time to do it on Friday, and they will be able to start faster on Saturday. I think it's more useful than configuring username/email for Git, which isn't actually useful to them yet.

They even have students deploy the throwaway app to Heroku - I wonder if that's information overload?

**Comment by gabebw** [comment]
I don't have a strong opinion about Heroku, but I really like having them choose their OS first instead of having them choose their OS on every page.

**Comment by decklin** [comment]
I want to take a shot at this again between now and June. We now have a Windows 10 test machine, which I am hanging on to, to make screenshots. (Gotta finally get my Mac on El Capitan I guess...)

**Comment by jgendrano** [comment]
I did upstream's Installfest and I agree that it was super clear save a few version discrepancies.

I think the instructions could be streamlined especially for Windows users. You had a lot of people flipping back and forth between the download pages and linear instructions. I ran into a few minor issues installing on an 8.1 computer as well which I made notes for. I may try Installfest (Boston version) on a Windows 7 computer to see if I run into the same issues before upgrading both of my Windows machines to 10.
